proseco/acq.py

```diff
@@ -178,6 +183,41 @@ def boxes_overlap(
     return 0
 
 
+def get_acq_candidates_mask(stars: "Table") -> np.ndarray:
+    """Get filter on ``stars`` for acceptable acquisition star candidates.
+
+    This does not include spatial filtering.
+
+    Parameters
+    ----------
+    stars : Table
+        Table of stars
+
+    Returns
+    -------
+    ok : np.array of bool
+        Mask of acceptable stars
+    """
+    # Allow for stars from proseco StarsTable.from_stars() with `mag` and `mag_err`,
+    # or native agasc.get_agasc_cone() stars table.
```

Comment:
Is this tested?

Reply:
Yes. Proseco supplies `stars` with `mag` and `mag_err` columns while `chandra_aca` (https://github.com/sot/chandra_aca/pull/170) plotting uses native `agasc` names `MAG` and `MAG_ACA_ERR`.

Reply:
So functionally tested vial the chandra_aca PR.

Reply:
Correct.  I updated the description.